kolibri/plugins/learn/assets/src/views/HybridLearningLessonCard.vue

```diff
@@ -82,7 +82,7 @@
     @extend %dropshadow-1dp;
 
     &:hover {
-      @extend %dropshadow-4dp;
+      @extend %dropshadow-2dp;
```

Comment:
Since the other "card" elements updated here have been changed to `6dp` ("HybridLearningContentCard", "ChannelCard"), this should also be `6dp` for consistency